src/start.sh

```diff
@@ -75,10 +75,13 @@ start() {
         sleep 0.5
     done
 
-    #  Wait until the FTL log contains the "FTL started" message before continuing
-    while ! grep -q '########## FTL started' /var/log/pihole/FTL.log; do
-        sleep 0.5
-    done
+    # Wait until the FTL log contains the "FTL started" message before continuing, timeout after 10 seconds
+    # exit if we do not find it
+    pihole-FTL wait-for '########## FTL started' /var/log/pihole/FTL.log 10 "0 > /dev/null"
+    if [ $? -ne 0 ]; then
+        echo "  [✗] FTL did not start"
```

Comment:
I'm just being pernickety at this point, but can we tack on something like ", stopping container" here. Not 100% on the wording, but i'd like just a little more verbosity in the sentence

```suggestion
        echo "  [✗] FTL did not start - stopping container"
```